types/parse5/v4/index.d.ts

```diff
@@ -0,0 +1,1081 @@
+// Type definitions for parse5 4.0
+// Project: https://github.com/inikulin/parse5
+// Definitions by:  Ivan Nikulin <https://github.com/inikulin>
```

Comment:
@inikulin I've added you here as a maintainer because I _believe_ you were the original author of these types?

There's no pressure to be a maintainer for them now, so I can remove your name here if you'd like.

Reply:
Thanks, I'm fine maintaining it.